sprocs/index.js

```diff
@@ -74,6 +74,9 @@ module.exports = {
             'config_select.sql',
             'users_select_or_insert.sql',
             'dump_to_csv.sql',
+            'calculate_assessment_question_stats_for_assessment.psql',
+            'calculate_assessment_question_stats.psql',
+            'calculate_stats_for_instance_questions.psql'
```

Comment:
Done. I haven't changed the actual function names though. Should I change those too?